packages/strapi-admin/admin/src/containers/Roles/ListPage/ListWrapper.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const ListWrapper = styled.div`
+  border-radius: 2px;
+  box-shadow: 0 2px 4px #e3e9f3;
+  background: white;
+  > div,
+  > div > div:last-of-type {
+    box-shadow: none;
+    border-radius: 2px;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+  p {
+    margin-bottom: 0;
+  }
+`;
```

Comment:
I take the webhook list wrapper style (It should be in the buffet List component to set the bottom list button). Already added to the front todo board.